projects/pt1/e2e_testing/main.py

```diff
@@ -93,6 +98,18 @@ def _get_argparse():
 def main():
     args = _get_argparse().parse_args()
 
+    ir_printer = logging.getLogger("ir_printer")
+    if args.print_ir:
+        print("WARNING: --print-ir is a work in progress feature.")
+        print("print-ir: Setting logging level to DEBUG and enabling IR printing.")
+        print("print-ir: This currently only affects the Linalg-on-Tensors and onnx configs.")
+        print("print-ir: Work in progress. See https://github.com/llvm/torch-mlir/issues/3172")
+        ir_printer.setLevel(logging.DEBUG)
+    else:
+        # disable logging
+        ir_printer.setLevel(logging.CRITICAL+1)
```

Comment:
Why is this CRITICAL+1 ?

Reply:
Was looking for the python idiom for completely disabling logging. This looks like the way to disable it.